mentors/rust/r.json

```
@@ -6,5 +6,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "An avid believer of the Feynman technique. Currently a professional trying to learn Rust and also teach it at the same time."
+  },
+  {
+    "github_username": "ronCYA",
+    "name": "Ron Au",
+    "link_text": "ronvoluted.com",
```

link_text should be something other than the link_url.  If the link_text wants to be the same as link_url then
```suggestion
    "link_text": null,
```

---

That makes sense, apologies for that. This has been ammended.